apps/frontend/src/utils/constants.ts

```diff
@@ -438,7 +436,7 @@ export const INITIATE_FIELDS = {
 			domainDepended: true,
 			options: {
 				agri: AGRI_SCENARIOS["search"].map((each) => each.scenario),
-				agrioutput:AGRI_OUTPUT_SCENARIOS["search"].map((each)=>each.scenario)
+				// agrioutput:AGRI_OUTPUT_SCENARIOS["search"].map((each)=>each.scenario)
```

Comment:
We are supposed to send the scenario when "initiating the request". Please understand that when "initiating" the request, the API call first goes to the Mock server. The Mock Server then constructs the Payload, signs it and then sends it to the NP's Buyer or Seller App. As such, on `/initiate` endpoints, we can take scenarios. But on endpoints which have the format `<domain>/bap/<action>` or `<domain>/bpp/<action`, we cannot take scenario or query.